src/mumble/MainWindow.cpp

```diff
@@ -3811,3 +3813,22 @@ void MainWindow::destroyUserInformation() {
 		}
 	}
 }
+
+void MainWindow::on_muteCuePopup_triggered() {
+	if (Global::get().s.muteCueShown || Global::get().inConfigUI)
```

Comment:
Sorry to bother you again with this, but I did not see it the first time around. Could you please add ``{`` and ``}`` around single line if statements?